### PR TITLE
[bitnami/kube-prometheus] Allow custom prometheus url in Thanos sidecar

### PR DESCRIPTION
**Pull request by EtienneBarbier**
**Description of the change**

Add possibility to set prometheus url in Thanos sidecar.

**Benefits**

We can specify prometheus url with pathPrefix.

**Possible drawbacks**

None

**Applicable issues**

None

**Additional information**

None

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/kube-prometheus/templates/prometheus/prometheus.yaml [hunk 1]
```diff
@@ -211,7 +211,11 @@ spec:
       imagePullPolicy: {{ .Values.prometheus.thanos.image.pullPolicy }}
       args:
         - sidecar
+        {{- if .Values.prometheus.thanos.prometheusUrlOverride }}
         - --prometheus.url=http://localhost:9090
+        {{- else }}
+        - --prometheus.url={{ .Values.prometheus.thanos.prometheusUrlOverride }}
+        {{- end }}
```
Comment (alemorcuq) [inline bitnami/kube-prometheus/templates/prometheus/prometheus.yaml:218]: You can use something like this instead:

```
--prometheus.url={{ default "http://localhost:9090" .Values.prometheus.thanos.prometheusUrl }}
```

**Review by alemorcuq — CHANGES_REQUESTED** [review]
Thanks for the PR! Please, see my comments below.

File: bitnami/kube-prometheus/README.md [hunk 2]
```diff
@@ -258,6 +258,7 @@ The following table lists the configurable parameters of the kube-prometheus cha
...
 | `prometheus.thanos.image.pullPolicy`                       | Thanos image pull policy                                                                                | `IfNotPresent`                                                                                                                          |
 | `prometheus.thanos.image.pullSecrets`                      | Specify docker-registry secret names as an array                                                        | `[]` (does not add image pull secrets to deployed pods)                                                                                 |
+| `prometheus.thanos.prometheusUrlOverride`                  | Specify custom prometheus url                                                                           | `""`                                                                                                                                    |
```
Comment (alemorcuq) [inline bitnami/kube-prometheus/README.md:261]: I think the parameter should be called `prometheusUrl` alone. No need for the `Override` part.

**Comment by EtienneBarbier** [comment]
@alemorcuq Thank you for theses improvements, I will push a new commit :)

**Review by alemorcuq — APPROVED** [review]
Thanks!

**Pull request merged**